src/plone/volto/locales/plone.volto.pot

```diff
@@ -195,10 +199,26 @@ msgstr ""
 msgid "URL of blocks-conversion-tool. The migration requires a service to run. See https://github.com/plone/blocks-conversion-tool for details."
 msgstr ""
 
-#: plone/volto/profiles.zcml:25
+#: plone/volto/profiles.zcml:33
 msgid "Uninstalls the plone.volto add-on."
 msgstr ""
 
+#: plone/volto/interfaces.py:22
+msgid "Used for rewriting URL's sent in the password reset e-mail by Plone."
```

Comment:
Same as earlier.

```suggestion
msgid "Used for rewriting URLs sent in the password reset email by Plone."
```

Reply:
@stevepiercy Done with the commit [0f76754](https://github.com/plone/plone.volto/pull/199/commits/0f767544179f5ff33b8a41739b6be485bb170c58)